src/components/TooltipNew/PopperTooltip.js

```diff
@@ -347,6 +361,7 @@ PopperTooltip.propTypes = {
    * If you are using `triggerActionType='managed'` event handler will call provided onClose prop.
    */
   closeOnOutsideClick: PropTypes.bool,
+  closeWithX: PropTypes.bool,
```

Comment:
I not going to complain about it this time, but it's important to add comments to props. If you add a comment to property it will be rendered in the docs.

Reply:
no problem I added some docs

also I added another way to close tooltip - with esc key (request by jo)